mobile/library/cc/engine_callbacks.cc

```diff
@@ -7,7 +7,9 @@ namespace {
 
 void c_on_engine_running(void* context) {
   auto engine_callbacks = *static_cast<EngineCallbacksSharedPtr*>(context);
-  engine_callbacks->on_engine_running();
+  std::function<void()> on_engine_running_cb = engine_callbacks->on_engine_running;
+  engine_callbacks->on_engine_running = {};
```

Comment:
ahh didn't realize you can clear out the lambda this way!  (i thought you had to set it to nullptr)